config.rb

```diff
@@ -0,0 +1,29 @@
+require 'compass/import-once/activate'
+# Require any additional compass plugins here.
+
+# Set this to the root of your project when deployed:
+http_path = "/"
+css_dir = "css"
+sass_dir = "sass"
+images_dir = "img"
+javascripts_dir = "js"
+
+output_style = :expand
```

Comment:
A couple of us set this to `:compressed` and use source maps to bridge the gap between the minified code and the original files. Check out sourcemaps in http://compass-style.org/help/documentation/configuration-reference/

Feel free to ping me if you have any questions.